tools/pot/docs/DefaultQuantizationUsage.md

```diff
@@ -11,29 +11,28 @@
 
 ![](./images/default_quantization_flow.png)
 
 The script should include three basic steps:
-1. Prepare data and dataset interface
-2. Select quantization parameters
-3. Define and run quantization process
+1. Prepare data and dataset interface.
+2. Select quantization parameters.
+3. Define and run quantization process.
 
 ## Prepare data and dataset interface
 In most cases, it is required to implement only `openvino.tools.pot.DataLoader` interface which allows acquiring data from a dataset and applying model-specific pre-processing providing access by index. Any implementation should override the following methods: 
 
-- `__len__()`, returns the size of the dataset
-- `__getitem__()`, provides access to the data by index in range of 0 to `len(self)`. It also can encapsulate the logic of model-specific pre-processing. The method should return data in the following format:
-   - `(data, annotation)`
-
-where `data` is the input that is passed to the model at inference so that it should be properly preprocessed. `data` can be either `numpy.array` object or dictionary, where the key is the name of the model input and value is `numpy.array` which corresponds to this input. Since `annotation` is not used by the Default Quantization method this object can be `None` in this case.
+- The `__len__()`, returns the size of the dataset.
+- The `__getitem__()`, provides access to the data by index in range of 0 to `len(self)`. It also can encapsulate the logic of model-specific pre-processing. This method should return data in `(data, annotation)` format, in which:
+   * The `data` is the input that is passed to the model at inference so that it should be properly preprocessed. It can be either `numpy.array` object or dictionary, where the key is the name of the model input and value is `numpy.array` which corresponds to this input.
+   * The `annotation` is not used by the Default Quantization method. Therfore, this object can be `None` in this case.
   
-You can wrap framework data loading classes by `openvino.tools.pot.DataLoader` interface which is usually straightforward. For example, `torch.utils.data.Dataset` has a similar interface as `openvino.tools.pot.DataLoader` so that its TorchVision implementations can be easily wrapped by POT API.
+Framework data loading classes can be wrapped by `openvino.tools.pot.DataLoader` interface which is usually straightforward. For example, `torch.utils.data.Dataset` has a similar interface as `openvino.tools.pot.DataLoader`, so that its TorchVision implementations can be easily wrapped by POT API.
```

Comment:
```suggestion
Framework data loading classes can be wrapped by the `openvino.tools.pot.DataLoader` interface which is usually straightforward. For example, the `torch.utils.data.Dataset` has a similar interface as the `openvino.tools.pot.DataLoader`, so that its TorchVision implementations can be easily wrapped by POT API.
```